lib/growthbook/context.rb

```diff
@@ -34,14 +34,19 @@ def initialize(options = {})
       @enabled = true
       @impressions = {}
 
-      options.each do |key, value|
-        case key.to_sym
+      options.transform_keys(&:to_sym).each do |key, value|
+        case key
         when :enabled
           @enabled = value
         when :attributes
           self.attributes = value
         when :url
           @url = value
+        when :encryption_key
```

Comment:
I prefer calling this `decryption_key` since it's used to decrypt an already encrypted payload.